config/prow/plugins.yaml

```diff
@@ -1579,6 +1579,10 @@ plugins:
     plugins:
     - assign  # Allow /assign and /cc
     - trigger # Allow people to configure CI jobs to /test
+    - label   # Allow people to label issues and PRs
```

Comment:
Do we need a set of restricted labels also? Because we have `priority` and `status` labels in containerd. For k8s I think certain labels are handled with restriction.

Reply:
It doesn't look like the `label` plugin handles `status` by default.  `priority` labels in containerd are mostly unused.  I figure we can always change this config later if it proves to be an issue?

Reply:
Yes. `status` is not handled by default.